docs/content/docs/test/Run.md

```diff
@@ -1,39 +1,111 @@
+- [Android Lint]({{< ref "/docs/checks/AndroidLint.md" >}})
+
+**ВАЖНО!** Тесты, почеменные как [`@Flaky`]({{< ref "/docs/test/FlakyTests.md" >}}) запускаются, но на результат прогона
+не влияют.
+
+За подробностями можно обратиться к конфигурации `fullCheck` в `avito/build.gradle`.
+
+### Регресс
+
+Регресс [автоматически запускается](http://links.k.avito.ru/OjE) каждый день со вторника по субботу в 5 и 8 утра, а
+также каждые 4 часа в понедельник и воскресенье. Результаты прогонов агрегируются в отчет по коммиту, с которого был
+запущен регресс, и упавшие тесты рассылаются ответственным юнитам в соответствующие чаты в Slack.
+
+Регресс включает в себя:
+- Юнит тесты
+- Инструментальные тесты:
+  - Компонентные
+  - E2E
+  - [Скриншотные]({{< ref "/docs/test/ScreenshotTesting.md" >}})
+  - Интеграционные (`@IntegrationTest`)
+  - Ручные (`@ManualTest`)
+- [Android Lint]({{< ref "/docs/checks/AndroidLint.md" >}})
+
+За подробностями можно обратиться к конфигурации `release` в `avito/build.gradle`.
+
+## Запуск тестов вручную
+
+### IDE
 
 Обрати внимание на [кастомные конфигурации]({{< ref "/docs/ide/Configurations.md" >}})
 
-### Robolectric
+#### Robolectric
 
 Просто запускаем стрелками Run напротив имени тестового класса или метода.
 
-Важно чтобы в настройках запуск был делегирован Gradle:
+#### Instrumentation
+
+Аналогично Robolectric тестам - просто запускаем тест средствами Android Studio.
+
+### Запуск всех E2E тестов
 
-**Preferences > Build, Execution, Deployment > Build Tools > Gradle**
+Для запуска всех E2E тестов воспользуйтесь Team City конфигураций [fullCheck](http://links.k.avito.ru/5u). В настройках
+запуска можно задать следующие параметры:
+- Ветку для запуска (*Changes -> Build branch*)
+- API URL, в который ходят тесты (*Parameters -> Test URL*)
 
-{{< tabs "robolectric" >}}
-{{< tab "AS 3.6+" >}}
-Build and run using: Gradle
-Run tests using: Gradle
-{{< /tab >}}
-{{< tab "AS 3.5" >}}
-- Delegate IDE build/run actions to Gradle
-- Run tests using: Gradle Test Runner
-{{< /tab >}}
-{{< /tabs >}}
+### Запуск тестов в CI с кастомными параметрами {#custom-run}
 
-### Instrumentation
+В Teamcity есть [конфигурация `instrumentationDynamic`](http://links.k.avito.ru/tmctAvitoAndroidInstrumentationDynamic)
+для запуска тестов со специфическим набором требований.
 
-#### Known issues
+Запускайте и вам будет предложено выбрать:
+- Ветку для запуска (*Changes -> Build branch*)
+- API URL, в который ходят тесты (*Parameters -> Test URL*)
+- Предустановленный набор тестов (*Parameters -> env.DYNAMIC_FILTER*)
+  - `ui` -- компонентные, E2E, скриншотные, интеграционные тесты
+  - `uiNoE2E` -- как `ui`, но без E2E тестов
+  - `regressionNoE2E` -- `uiNoE2E` + ручные тесты
+  - `regression` -- `ui` + ручные тесты
+  - `performanceNoE2E` -- только компонентные перформанс тесты
+  - `performance` -- только функциональные перформанс тесты
+- Фильтр тестов в формате `[+ имя класса [+ имя тестового метода ]]` (*Parameters -> Package prefix*)
```

Comment:
I'm not sure, is it clear that I need to write <testClass>.<testName> separated by `.`?

Reply:
I've replaced it with a concrete example `[com.test.Foo[.someTest]]`